components/Slider/tests/unit/SliderTests.m

```diff
@@ -1103,7 +1098,7 @@ - (void)testAccessibilityIncrementDiscreteSlider {
     CGFloat stepValue = (self.slider.maximumValue - self.slider.minimumValue) /
         (self.slider.numberOfDiscreteValues - 1);
     CGFloat expectedValue = self.slider.minimumValue + stepValue;
-    XCTAssertEqualWithAccuracy(self.slider.value, expectedValue, 0.0001,
+    XCTAssertEqualWithAccuracy(self.slider.value, expectedValue, kEpsilonAccuracy,
```

Comment:
Why the change from `0.001` to `kEpsilonAccuracy`, I thought `0.001` was okay in tests?

Reply:
I was just reusing the accuracy value that was already in-use within the file. It didn't seem like the difference (0.0001 to 0.001) would have any effect on the calculations being performed.